package.json

```diff
@@ -3,13 +3,16 @@
   "version": "0.38.2",
   "description": "Free-range app utility for IFRAME-based FRAs",
   "type": "module",
-  "exports": "./index.js",
+  "exports": {
+    "./client/client.js": "./client/client.js",
```

Comment:
I could make this something like `"./client/client": "./client/client.js"` instead to make the import statements a bit prettier, but I don't have any opinion on whether that's a net improvement or not.

Reply:
I think all of the web stuff and new node.js loaders are always-include-the-extension, so this seems okay.

Reply:
Consider `./client.js` to aligh with the old usage of `ifrau/client`?

Reply:
Yeah, that was my line of thinking for including it by default. At the very least, our Lit web components now enforce this in preparation for Lit 2.0.